styles_src/default.scss

```diff
@@ -133,6 +133,21 @@ $party-margin: 2px;
     }
   }
 
+  &-available-seats {
+    width:100%; 
+    text-align:right;
+    border-bottom: 1px solid #000; 
+    line-height:0.1em!important;
+    margin: 20px 0 16px 0;
+    
```

Comment:
You can also add the block below to this block

```scss
&-text  {
...
}
```

Because its essentially just a nested block of `-available-seats-...`